### PR TITLE
Feat 20 register docker

### LINKED ISSUE #20: Deploy docker image to GitHub Container registry
# User story
As a developer, I want to deploy the docker image to a registry, so I can easily deploy it as a service.

# Acceptance criteria
- [ ] When a PR is merged into main, then a docker image is build and register to GitHub's container registry that has a namespace like: `https://ghcr.io.`

# Sprint Ready Checklist 
- [ ] 1. Acceptance criteria defined 
- [ ] 2. Team understands acceptance criteria 
- [ ] 3. Team has defined solution / steps to satisfy acceptance criteria 
- [ ] 4. Acceptance criteria is verifiable / testable 
- [ ] 5. External / 3rd Party dependencies identified
- [ ] 6. Ticket is prioritized and sized

# Notes
*Add any helpful notes here.*

### LINKED ISSUE #21: Update Issues Template
# User story
As a developer, I want the sprint checklist checkboxes swapped with the numbers, so I can easily check them off in GitHub Project cards.

# Acceptance criteria
- [ ] The issues template should use the Sprint Ready Checklist like below:

# Sprint Ready Checklist 
- [ ] 1. Acceptance criteria defined 
- [ ] 2. Team understands acceptance criteria 
- [ ] 3. Team has defined solution / steps to satisfy acceptance criteria 
- [ ] 4. Acceptance criteria is verifiable / testable 
- [ ] 5. External / 3rd Party dependencies identified
- [ ] 6. Ticket is prioritized and sized

# Notes
*Add any helpful notes here.*

### PR DESCRIPTION
**Pull request by jtyoung84**
Closes #20
Closes #21

- Tested on a forked repo: https://github.com/jtyoung84/aind-metadata-service
- Updates issue template
- Updates unit test to handle change in aind-data-schema repo

### CODE REVIEW

**Comment by jtyoung84** [comment]
A few people requested to be tagged to review this pr. @aricAI I'll merge it once you approve it.

**Review by aricAI — CHANGES_REQUESTED** [review]
in publish.yml lines 17 and 33, set-output is deprecated. https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

echo "version=$new_tag" >> $GITHUB_OUTPUT

**Comment by jtyoung84** [comment]
@dyf Should the owner by alleninstitute or allenneuraldynamics? The last two lines of the publish.yml file.

**Review by camilolaiton — APPROVED** [review]

**Review by dyf — APPROVED** [review]

**Review by aricAI — APPROVED** [review]

**Pull request merged**
